cedar-policy-core/src/evaluator/err.rs

```diff
@@ -374,13 +392,21 @@ pub mod evaluation_errors {
         impl_diagnostic_from_source_loc_field!();
 
         fn help<'a>(&'a self) -> Option<Box<dyn std::fmt::Display + 'a>> {
-            if self.available_attrs.is_empty() {
-                Some(Box::new("entity does not have any attributes"))
-            } else {
-                Some(Box::new(format!(
-                    "available attributes: {:?}",
-                    self.available_attrs
-                )))
+            match &self.available_attrs {
+                Some(attrs) => {
+                    if attrs.is_empty() {
+                        Some(Box::new("entity does not have any attributes"))
+                    } else {
+                        let mut attrs = attrs.iter().collect::<Vec<_>>();
+                        attrs.sort();
+                        Some(Box::new(format!("Available attributes: {:?}", attrs)))
```

Comment:
I think this needs to check whether `self.total_attrs < TOO_MANY_ATTRS` and if there's too many, say "(and `n` other attributes)" where `n` is `total_attrs - TOO_MANY_ATTRS`

Reply:
No, there's an invariant that's thread through here that's not made clear. If `available_attrs` is `Some` then it contains all of the attrs, as we can't sort the entire attr list if it's too big

Reply:
I'd just make `available_attrs` something more verbose then. like `first_five_available_attrs`